src/pages/CloudDeployProgress.js

```diff
@@ -44,7 +44,7 @@ const PLAYBOOK_STEPS = [
   },
   {
     label: translate('deploy.progress.predeployment'),
-    playbooks: ['dayzero-pre-deployment.yml', ]
+    playbooks: ['installui-pre-deployment.yml', ]
```

Comment:
It is odd that this code does not make use of the constants defined above. It should. I know that this duplication was not introduced by this change, but it would be nice to clean it up here.